src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/jpa/entity/ActionPlanAppointment.kt

```diff
@@ -16,6 +19,9 @@ data class ActionPlanAppointment(
 
   // Attributes
   @NotNull val sessionNumber: Int,
+  @Enumerated(EnumType.STRING)
+  var attended: SessionAttendance? = null,
+  var additionalAttendanceInformation: String? = null,
```

Comment:
💭 Providers have 1 working day (24 hours?) to mark attendance after an elapsed appointment.
Do we capture the time this information is submitted somewhere?